js/private/js_binary.bzl

```diff
@@ -168,6 +169,19 @@ _ATTRS = {
         """,
         default = True,
     ),
+    "copy_data_to_bin": attr.bool(
+        doc = """When True, data files and the entry_point file are copied to the Bazel output tree before being passed
+        as inputs to runfiles.
+
+        This default to False. It should only be needed if the program manages to skirt the node js patches and leave
```

Comment:
"node js" => "Node.js"?

I've noticed we have a mix of those and just "node" throughout or docs. We should probably try to standardize those but I don't think "node js" adds any value to "node" (while "Node.js" seems like a more formal way of saying "node"? 🤷).

Reply:
Whoops. Was supposed to be "node fs patches"